lib/github-pages.rb

```diff
@@ -35,9 +35,22 @@ def self.gems
   # Versions used by GitHub Pages, including github-pages gem and ruby version
   # Useful for programmatically querying for the current-running version
   def self.versions
-    gems.merge({
-      "github-pages" => VERSION.to_s,
-      "ruby"         => RUBY_VERSION
-    })
+    gems.merge deps_version_report
+  end
+
+  def self.deps_version_report
+    require 'html/pipeline/version'
+    require 'sass/version'
+    require 'safe_yaml/version'
+
+    {
+      "ruby" => RUBY_VERSION,
+
+      # Gem versions we're curious about
+      "github-pages"  => VERSION.to_s,
+      "html-pipeline" => HTML::Pipeline::VERSION,
```

Comment:
The motivation for reporting on these is that as these dependencies may differ locally versus the server. HTML Pipeline changed a bunch between 1.8 and 1.9 and I remember having trouble because of this, but I didn't have the Gemfile.lock for Pages, so I was stuck.